doc/source/tutorials/elastic_networks.rst

```diff
@@ -60,7 +60,7 @@ The strength of the elastic bond can be tuned with distance using an exponential
 which uses the ``-ea`` and ``-ep`` flags as input parameters:
 
 
-``decay = exp^{(- f * ((x - l) ^p)}``
+$$decay = e^{(- f * ((x - l) ^ p)}$$
```

Comment:
I got the syntax wrong, should be:
```suggestion
.. math::
    :label: decay
    decay = e^{(- f * ((x - l) ^ p)}
```

You can refer to this equation then using ```:eq:`decay` ```.
See https://ikerdocs-sphinx.readthedocs.io/syntax/math.html

This may also need (some) changes to `doc/source/conf.py`: https://www.sphinx-doc.org/en/master/usage/extensions/math.html and https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-math.
mathjax looks fine to me.